### ISSUE TITLE
Clearing cache should be the obvious first thing to do

### ISSUE THREAD

**Issue by Biep** [vector-im/riot-web#9321]
Clicking on one's user name (top left) should show 'Solve problems' separately from 'Settings' and the like - and it should bring up a panel that offers 'Clear cache' as the first option, with some text telling people to try that first, and reassuring them that it won't make them lose information.

**Comment by turt2live** [comment]
tbh I'd rather we fixed the problems than spent time trying to engineer clear cache buttons into the UI, even if fixing the problems takes longer.

**Comment by Biep** [comment]
I fully agree, but the one doesn't need to exclude the other, and moving the "clear cache" button around seems a trivial change.
If we lose popularity because of the bugs we may not build up the critical momentum we need.

**Comment by lampholder** [comment]
I think something like this is worth considering - I've used software long enough that I might go hunting in settings for some troubleshooting/assistance, but it's not the most logical place for it to be.

I reckon it's worth thinking about:
- how discoverable the lifesaver is in the bottom left (we're going to add analytics to track usage)
- whether we can provide a better flow for users experiencing problems, along the lines of:
     - having difficulties? some problems can be solve by refreshing the cache - click here to submit logs and clear the cache (click here to just clear the cache)
